sql/src/test/java/io/crate/breaker/RowAccountingWithEstimatorsTest.java

```diff
@@ -64,6 +65,19 @@ public void testCircuitBreakingWorks() throws Exception {
         RowGenerator.range(0, 3).forEach(rowAccounting::accountForAndMaybeBreak);
     }
 
+    @Test
+    public void testRowCellsAccountingCircuitBreakingWorks() throws Exception {
+        RowCellsAccountingWithEstimators rowAccounting = new RowCellsAccountingWithEstimators(Collections.singletonList(DataTypes.INTEGER),
```

Comment:
style nitpick: Think it would be easier to read (e.g. in github diffs) if the first argument was also in a new line to start the indentation further to the left.